### PR TITLE
Remove backend code

### PR DESCRIPTION
**Pull request by andresmgot**
Leave this repository as frontend only.

Issue: https://github.com/grafana/grafana-aws-sdk/issues/26

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["@grafana/eslint-config"],
+  "root": true,
+  "plugins": ["no-only-tests", "@emotion", "lodash"],
+  "rules": {
+    "no-only-tests/no-only-tests": "error",
+    "react/prop-types": "off",
+    "@emotion/jsx-import": "error",
+    "lodash/import-scope": [2, "member"]
```
Comment (sarahzinger) [inline .eslintrc:9]: just curious what inspired the need to extend what we have? Seems like we'd want to keep things more or less in line with whatever the defaults right?

**Review by sarahzinger — APPROVED** [review]
🎉

File: Magefile.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func Drone() error {
 		return err
 	}
 
-	if err := sh.RunV("drone", "--server", "https://drone.grafana.net", "sign", "--save", "grafana/grafana-plugin-sdk-go"); err != nil {
+	if err := sh.RunV("drone", "--server", "https://drone.grafana.net", "sign", "--save", "grafana/grafana-aws-sdk-react"); err != nil {
```
Comment (sunker) [inline Magefile.go:35]: Do we need to have this file for Drone?

**Review by sunker — APPROVED** [review]
Have you thought about publishing? I think [this](https://github.com/grafana/eslint-config-grafana/pull/10/files) solution is pretty neat. It published a new version every time a PR is merged to main and it includes changes to the version prop.

File: .eslintrc (hunk 1 above)
Comment (andresmgot) [reply .eslintrc:9]: I just copied this from the root of grafana/grafana but I guess it's not really that needed. I can just use the default.

File: Magefile.go (hunk 2 above)
Comment (andresmgot) [reply Magefile.go:35]: Not really, it was just out of convenience to not having to remember the whole drone command but I can just add it to the docs.

**Comment by andresmgot** [comment]
> Have you thought about publishing? I think [this](https://github.com/grafana/eslint-config-grafana/pull/10/files) solution is pretty neat. It published a new version every time a PR is merged to main and it includes changes to the version prop.

Thanks for the pointer! Yes, I was thinking on adding that when releasing but I can add it to this PR.

**Pull request merged**